docs/content/integrations/spark.mdx

```diff
@@ -1,62 +1,60 @@
 
-There are two approaches to writing Dagster ops that invoke Spark computations:
+There are two approaches to writing Dagster assets and ops that invoke Spark computations:
 
-## Op body submits job
+## Asset body submits Spark job
 
-With this approach, the code inside the op submits a Spark job to an external system like Databricks or EMR, usually pointing to a jar or zip of Python files that contain the actual Spark data transformations and actions.
+With this approach, the code inside the asset definition submits a Spark job to an external system like Databricks or EMR, usually pointing to a jar or zip of Python files that contain the actual Spark data transformations and actions.
 
-If you want to use this approach to run a Spark job on Databricks, <PyObject module='dagster_databricks' object='create_databricks_run_now_op' /> or <PyObject module='dagster_databricks' object='create_databricks_submit_run_op' /> help create ops that submit a Spark job using the Databricks REST API.
+If you want to run a Spark job against YARN or a Spark Standalone cluster, the code inside your asset can issue a shell command invokes `spark-submit`.
 
-If you want to run a Spark job against YARN or a Spark Standalone cluster, you can use <PyObject module='dagster_shell' object='create_shell_command_op' /> to create an op that invokes `spark-submit`.
+This is the easiest approach for migrating existing Spark jobs, and it's the only approach that works for Spark jobs written in Java or Scala. The downside is that it loses out on some of the benefits of Dagster - the implementation of each asset is bound to the execution environment, so you can't run your Spark transformations without relying on external infrastructure. Writing unit tests is cumbersome.
 
-This is the easiest approach for migrating existing Spark jobs, and it's the only approach that works for Spark jobs written in Java or Scala. The downside is that it loses out on some of the benefits of Dagster - the implementation of each op is bound to the execution environment, so you can't run your Spark transformations without relying on external infrastructure. Writing unit tests is cumbersome.
+## Asset accepts and produces DataFrames or RDDs
 
-## Op accepts and produces DataFrames or RDDs
+With this approach, the code inside the asset definition consists of pure logical data transformations on Spark DataFrames or RDDs. The `asset`-decorated function accepts `DataFrames` as parameters and returns `DataFrames` when it completes. An [IO manager](/concepts/io-management/io-managers) handles writing and reading the DataFrames to and from persistent storage. The [Running PySpark code in assets](#running-pyspark-code-in-assets) example below shows what this looks like.
```

Comment:
> The `asset`-decorated function

Why use code formatting for asset over leaving it plain or working in the PyObject component for `@asset`?

Reply:
Good point. I fixed this to use PyObject.